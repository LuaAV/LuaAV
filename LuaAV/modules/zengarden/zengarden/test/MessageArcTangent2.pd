#N canvas 695 89 450 300 10;
#X obj 32 17 loadbang;
#X obj 32 194 print;
#X obj 101 17 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 32 99 3.7;
#X obj 32 159 atan2;
#X msg 199 99 0;
#X obj 32 54 t b b b b b b b b;
#X msg 156 99 44;
#X msg 117 99 -2;
#X msg 237 99 -2;
#X msg 280 99 -4 3;
#X msg 324 99 -5.5;
#X msg 68 99 14.3;
#X connect 0 0 6 0;
#X connect 2 0 6 0;
#X connect 3 0 4 0;
#X connect 4 0 1 0;
#X connect 5 0 4 0;
#X connect 6 0 3 0;
#X connect 6 1 12 0;
#X connect 6 2 8 0;
#X connect 6 3 7 0;
#X connect 6 4 5 0;
#X connect 6 5 9 0;
#X connect 6 6 10 0;
#X connect 6 7 11 0;
#X connect 7 0 4 1;
#X connect 8 0 4 0;
#X connect 9 0 4 1;
#X connect 10 0 4 0;
#X connect 11 0 4 1;
#X connect 12 0 4 1;
