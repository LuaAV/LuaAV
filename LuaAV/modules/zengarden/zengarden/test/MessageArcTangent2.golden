[@ 0.000ms] print: -0.927295
[@ 0.000ms] print: 3.14159
[@ 0.000ms] print: -0.0454233
[@ 0.000ms] print: 0.253189
